src/frontend/src/handler/util.rs

```diff
@@ -257,6 +260,26 @@ pub fn get_connection_name(with_properties: &BTreeMap<String, String>) -> Option
         .map(|s| s.to_lowercase())
 }
 
+/// `alter_definition_add_column` adds a new column to the definition of the relation.
+pub fn alter_definition_add_column(definition: &str, column: ColumnDef) -> RwResult<String> {
```

Comment:
Will this be utilized in other places? If not I guess we can inline it in `alter_source_column`. 😂